src/vs/editor/contrib/multicursor/browser/multicursor.ts

```diff
@@ -667,14 +668,18 @@ export abstract class MultiCursorSelectionControllerAction extends EditorAction
 		if (!multiCursorController) {
 			return;
 		}
-		const findController = CommonFindController.get(editor);
-		if (!findController) {
-			return;
-		}
 		const viewModel = editor._getViewModel();
 		if (viewModel) {
 			const previousCursorState = viewModel.getCursorStates();
-			this._run(multiCursorController, findController);
+			const findController = CommonFindController.get(editor);
+			if (findController) {
+				this._run(multiCursorController, findController);
+			} else {
+				const newFindController = accessor.get(IInstantiationService).createInstance(CommonFindController, editor);
```

Comment:
We have a dozen of find commands that can open find widget in editor and it's not ergonomic to override those keybindings in notebook so here I remove the find controller contrib from Monaco editors in notebook. The catch is multi cursor will not work as there is not find controller bind to an editor. Here I tried to create find controller instance when it's missing and dispose it after use, notebook users then pay price of creating find controller each time when using multi cursor commands.

Reply:
cc @alexdima